.github/workflows/deploy-nightly-playground.yml

```diff
@@ -0,0 +1,106 @@
+name: Nightly Playground Validate and Deploy
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 0 * * *'
+
+
+env:
+  DIST_VERSION: 2.11.0
+
+jobs:
+  set-os-osd-urls:
+    runs-on: ubuntu-latest
+    outputs:
+      OPENSEARCH_URL: ${{ steps.set-env.outputs.OPENSEARCH_URL }}
+      OPENSEARCH_DASHBOARDS_URL: ${{ steps.set-env.outputs.OPENSEARCH_DASHBOARDS_URL }}
+      OPENSEARCH_BUILD_ID: ${{ steps.set-env.outputs.OPENSEARCH_BUILD_ID }}
+      OPENSEARCH_DASHBOARDS_BUILD_ID: ${{ steps.set-env.outputs.OPENSEARCH_DASHBOARD_BUILD_ID }}
+      OPENSEARCH_DIST_MANIFEST_URL: ${{ steps.set-env.outputs.OPENSEARCH_DIST_MANIFEST_URL }}
+      OPENSEARCH_DASHBOARDS_DIST_MANIFEST_URL: ${{ steps.set-env.outputs.OPENSEARCH_DASHBOARDS_DIST_MANIFEST_URL }}
+    steps:
+      - name: Set Env variables
+        id: set-env
+        run: |
+          opensearch_url_latest=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${DIST_VERSION}/latest/linux/x64/tar/dist/opensearch/opensearch-${DIST_VERSION}-linux-x64.tar.gz
```

Comment:
This can be fetched from dist yml as well `yq .build.location`.

Reply:
Done! Thanks!